<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InfoView</class>
 <widget class="QWidget" name="InfoView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>460</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="API_Title">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>450</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;性质特点&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="submain_Title">
   <property name="geometry">
    <rect>
     <x>82</x>
     <y>100</y>
     <width>300</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;get_delta_uniformity&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="main_Title">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>40</y>
     <width>250</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;差分均匀度&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="space_line" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>340</width>
     <height>1</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="api_line_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>490</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string> · FileMove</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="api_line_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>590</y>
     <width>300</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string> · FileMove</string>
   </property>
  </widget>
  <widget class="QLabel" name="api_line_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>590</y>
     <width>300</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string> · FileMove</string>
   </property>
  </widget>
  <widget class="QLabel" name="api_line_4">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>600</y>
     <width>300</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string> · FileMove</string>
   </property>
  </widget>
  <widget class="QLabel" name="v1_img">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>60</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="v1_mainLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;操作1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="v1_subLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:18pt;&quot;&gt;原因分析1&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="v2_subLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
   <property name="text">
    <string>原因分析2</string>
   </property>
  </widget>
  <widget class="QLabel" name="v2_mainLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>操作2</string>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="v3_img">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>320</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="v3_subLabel">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>340</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;原因分析3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="v3_mainLabel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>290</y>
     <width>361</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;操作3&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="v2_img">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>40</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
